Projects/Embark/Sources/Embark.swift

```diff
@@ -38,11 +38,25 @@ extension Embark: Presentable {
         let scrollView = FormScrollView()
         scrollView.backgroundColor = .brand(.primaryBackground())
         let form = FormView()
-        form.dynamicStyle = DynamicFormStyle.default.restyled { (style: inout FormStyle) in
-            style.insets = .zero
-        }
+
         bag += viewController.install(form, options: [], scrollView: scrollView) { scrollView in
             scrollView.alwaysBounceVertical = false
+
+            bag += scrollView
```

Comment:
you probably need to do `combineLatest(trait, contentSize)` else you are doing `when trait collection changes also take contentSize` but you probably wanna do `when trait or contentSize change do`